src/H5Tdbg.c

```diff
@@ -182,6 +182,31 @@ H5T_debug(const H5T_t *dt, FILE *stream)
             s1 = "struct";
             break;
 
+        case H5T_REFERENCE:
```

Comment:
Added in missing reference datatype debugging code while adding the new code for complex number datatypes below.